webapp/utils/utils.jsx

```diff
@@ -1344,3 +1349,39 @@ export function isEmptyObject(object) {
 export function updateWindowDimensions(component) {
     component.setState({width: window.innerWidth, height: window.innerHeight});
 }
+
+export function postMessageHtmlToComponent(html, AtMentionComponent, usernameMap) {
+    const parser = new Parser();
+    const attrib = 'data-mention';
+    const processNodeDefinitions = new ProcessNodeDefinitions(React);
+
+    function isValidNode() {
+        return true;
+    }
+
+    const processingInstructions = [
+        {
+            replaceChildren: true,
+            shouldProcessNode: (node) => node.attribs && node.attribs[attrib] && usernameMap.hasOwnProperty(node.attribs[attrib]),
+            processNode: (node) => {
+                const username = node.attribs[attrib];
+                return atMentionComponent(AtMentionComponent, usernameMap[username], username);
```

Comment:
I'm pretty sure you can actually make this like
```
return (
    <AtMentionComponent
        user={usernameMap[username]}
        username={username}
    />
);
```
to make it so that we don't need that other `atMentionComponent` function